package.json

```diff
@@ -22,7 +22,9 @@
   },
   "dependencies": {
     "atom-linter": "10.0.0",
-    "atom-package-deps": "5.0.0"
+    "atom-package-deps": "5.0.0",
+    "request": "^2.81.0",
+    "request-promise": "^4.1.1"
```

Comment:
If this is being used, I would prefer `request-promise-native`. There's no reason to bring in the Bluebird based version unless it's _absolutely_ necessary.